plugins/autowarmth.koplugin/main.lua

```diff
@@ -324,6 +328,11 @@ function AutoWarmth:scheduleNextWarmthChange(time_s, search_pos, from_resume)
         if next_sched_time_s > 0 then
             -- This setWarmth will call scheduleNextWarmthChange which will schedule setWarmth again.
             UIManager:scheduleIn(next_sched_time_s, self.setWarmth, self, self.sched_warmths[self.sched_warmth_index], true)
+        else
+            -- ToDo: check if a schedule has to be in 1 sec or maybe later lets say at
```

Comment:
You probably mean before merging, but in case not make it `--- @todo`.

Reply:
Yes, as this is a draft PR, I meant I will rule this special case out before merging. (This might be a point in corner cases or not dunno now.)

Tust to be sure, I thought:
-- todo won't pass CI, so a developer won`t forget it, and others **_do_** see this.
--@todo is for later optimizations, so this is a "don't forget", "do it later" hint.

Am I right with these assumptions?

But anyway thanks for your care to point this out.

Reply:
Yup, well, I don't think the first is typical but regardless. ^^